pages/apim/3.x/v4-beta/v4-beta-new-policy-execution-engine-introduction.adoc

```diff
@@ -0,0 +1,32 @@
+[[v4-beta-new-policy-execution-engine-introduction]]
+= V4 policy execution engine (V4 BETA)
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/v4_new_policy_execution_engine_introduction.html
+:page-folder: apim/v4-beta
+:page-layout: apim3x
+
+[label label-version]#New in version 3.20.0#
+[label label-version]#BETA release#
+
+Introduced in release 3.19.0 of the Gravitee API Management platform, there is a beta version of a new policy execution engine that enables an improved execution flow for synchronous APIs and supports event-driven policy execution for asynchronous APIs. It is based on a modern and fully reactive architecture designed to address a number of challenges Gravitee users have been facing with the existing policy execution engine, available with the Gravitee's V2 API definition specification used in versions prior to APIM 3.19.x.
+
+The new policy execution engine builds on the new V4 BETA API definition foundation of Gravitee's event-native platform, which adds features such as native support for Pub/Sub (Publish-Subscribe) design and enabling policies at message level. The new V4 BETA event-native API definition makes it possible for the Gateway to “know” what is necessary in order for it to handle asynchronous API use cases.
```

Comment:
I feel there is something missing in "The new policy execution engine builds on the new V4 BETA API definition foundation of Gravitee's event-native platform".

Reply:
I have simplified this sentence a bit.